src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -827,10 +857,24 @@ internal static Exception TransactionZombied(DbTransaction obj)
         internal const float FailoverTimeoutStep = 0.08F;    // fraction of timeout to use for fast failover connections
 
         // security issue, don't rely upon public static readonly values - AS/URT 109635
-        internal static readonly String StrEmpty = ""; // String.Empty
+        internal static readonly string StrEmpty = ""; // String.Empty
```

Comment:
Actually, the above comment seems to imply that using String.Empty would be a security issue. Does that make any sense to you @stephentoub ?